keel-actuator/src/main/kotlin/com/netflix/spinnaker/keel/actuation/ResourcePersister.kt

```diff
@@ -75,8 +74,7 @@ class ResourcePersister(
       .also {
         log.debug("Creating $it")
         resourceRepository.store(it)
-        resourceRepository.appendHistory(ResourceCreated(it, clock))
-        publisher.publishEvent(CreateEvent(it.name))
+        publisher.publishEvent(ResourceCreated(it, clock))
```

Comment:
We already had this `ResourceCreated` event and `CreateEvent` seemed to just duplicate it. Listeners can just use the same event type.